assets/wpuf/js/frontend-form.js

```diff
@@ -11,6 +11,20 @@
         });
     };
 
+    $.fn.extend({
+        /**

```

Comment:
Spacing of `/**`

Should look like this:
```js
/**
 * Short description. (use period)
 */
```

Reply:
Is this the method we're duplicating, both on the frontend and backend? If so, maybe a `@todo` comment mentioning this and we should fix it one day.

Reply:
@bwmarkle  This should be removed and the `@todo` should be added to https://github.com/BoldGrid/weforms/pull/89